packages/editor/src/plugins/image-gallery/components/add-images.tsx

```diff
@@ -1,11 +1,49 @@
 import IconChoose from '@editor/editor-ui/assets/plugin-icons/image-gallery/icon-layered-images.svg'
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+import { v4 as uuidv4 } from 'uuid'
 
-export function AddImages() {
+import type { ImageGalleryProps } from '..'
+
+export function createImagePlugin(url: string, caption: string) {
+  return {
+    plugin: EditorPluginType.Image,
+    state: {
+      src: url,
+      caption: {
+        plugin: EditorPluginType.Text,
+        state: [
+          {
+            type: 'p',
+            children: [
+              {
+                text: caption,
+              },
+            ],
+          },
+        ],
+        id: uuidv4(),
+      },
+    },
+    id: uuidv4(),
+  }
+}
```

Comment:
Move this into utils and name it `createEmptyImagePlugin`

Reply:
Done :heavy_check_mark: